### PR TITLE
First attempt at using rich-codex

### PR DESCRIPTION
**Pull request by ewels**
Automated screen grabs of command outputs for the documentation: https://github.com/ewels/rich-codex

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

**Comment by ewels** [comment]
Ok, so all of the images show the same thing:



But still, it's making images! 🎉

**Comment by ewels** [comment]
Yay, now we're getting much prettier (and varying) errors! ✨

**Comment by maxulysse** [comment]
> > What is this magic?
> 
> 😁 🧙🏻
> 
> https://twitter.com/tallphil/status/1545043723850895360?s=20&t=-7Gkg2drGOn_M_da_WeyGw
> 
> https://ewels.github.io/rich-codex/

Bioinformatician united

File: README.md [hunk 1]
```diff
@@ -370,73 +279,20 @@ The `nf-core download` command will download both the pipeline code and the [ins
...
-          Pipeline release: '3.0'
-          Pull containers: 'singularity'
-          Using $NXF_SINGULARITY_CACHEDIR': /path/to/demo/cachedir
-          Output directory: 'nf-core-rnaseq-3.0'
-INFO     Downloading workflow files from GitHub
-INFO     Downloading centralised configs from GitHub
-INFO     Found 29 containers
-Downloading singularity images ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% • 29/29 completed
-```
+![`ORDER=01 nf-core download rnaseq -r 3.0 --outdir nf-core-rnaseq -x none -c none`](docs/images/nf-core-download.svg)
```
Comment (mashehu) [inline README.md:282]: The `ORDER` prefixes can be removed as soon as https://github.com/ewels/rich-codex/issues/32 is fixed or we use `fake_commands` instead.

Comment (ewels) [reply README.md:282]: Now fixed and released ✅

**Comment by mashehu** [comment]
[rich-codex #1662](https://trello.com/c/RYeQtmvQ/1-rich-codex-1662)

**Comment by ewels** [comment]
Closing in favour of https://github.com/nf-core/tools/pull/1764

**Pull request closed without merging**